packages/design-system/src/deprecatedTokens/animations.tokens.js

```diff
@@ -1,12 +0,0 @@
-import { css, keyframes } from 'styled-components';
```

Comment:
about all deprecated things, did you check if it's used somewhere before?

Reply:
no, at least not everywhere, they are deprecated since #4198. This is the purpose of this PR to remove them.
I m almost sure no one use theses tokens but design-tokens is already quite used.